website/layout/AutodocsLayout.js

```diff
@@ -654,7 +654,7 @@ var Method = React.createClass({
           </span> || ''}
           {this.props.name}
           <span className="methodType">
-            ({this.props.params && this.props.params.length && this.props.params
+            ({this.props.params && (this.props.params.length !== 0) && this.props.params
```

Comment:
@mkonicek 
The point here I guess is to check whether `params` is an Array (by checking whether length exists).
If `params` is 100% array, then this length check can be simplily removed. If length is 0, map function will return an empty array, which has no problem in JSX.